lib/presentation/ui/occurrences/occurrences_item.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:vost/domain/models/occurrence_model.dart';
+import 'package:vost/presentation/assets/text_styles.dart';
+
+class OccurrencesItem extends StatefulWidget {
```

Comment:
Since you don't have any state being changed in this widget, can it be `Stateless`?

Reply:
I created as a StatefulWidget because we need to set the favorite, I don't know if it's the good place to do that, so I need to change to Stateless?